lib/zold/score.rb

```diff
@@ -35,6 +36,9 @@ class Score < Dry::Struct
     # Default strength for the entire system, in production mode.
     STRENGTH = 6
 
+    # Default number of cores used for score calculation
+    CORES = 2
```

Comment:
I don't think this is the right place to indicate how many cores we want to use, it seemts too technical to me.

My opinion is to move this constant to score_ext.rb

Reply:
Ok. Will change that soon. But I'll be on vacation for the next 10days.